components/dashboard/src/Pagination/Pagination.tsx

```diff
@@ -29,23 +29,23 @@ function Pagination(props: PaginationProps) {
     };
     const getClassnames = (pageNumber: string | number) => {
         if (pageNumber === currentPage) {
-            return "text-gray-500 w-8 text-center rounded-md hover:bg-gray-50 bg-gray-100 disabled pointer-events-none";
+            return "font-semibold text-gray-500 dark:text-gray-400 max-h-9 max-w-8 flex items-center justify-center rounded-md hover:bg-gray-50 dark:hover:bg-gray-800 dark:bg-gray-700 bg-gray-100 disabled pointer-events-none px-3 py-2";
         }
         if (pageNumber === "...") {
-            return "text-gray-500 w-8 text-center rounded-md hover:bg-gray-50 disabled pointer-events-none";
+            return "font-semibold text-gray-500 dark:text-gray-400 max-h-9 max-w-8 flex items-center justify-center rounded-md hover:bg-gray-50 dark:hover:bg-gray-800 disabled pointer-events-none px-3 py-2";
         }
-        return "text-gray-500 w-8 text-center rounded-md hover:bg-gray-50 cursor-pointer";
+        return "font-semibold text-gray-500 dark:text-gray-400 max-h-9 max-w-8 flex items-center justify-center rounded-md hover:bg-gray-50 dark:hover:bg-gray-800 cursor-pointer px-3 py-2";
     };
 
     return (
         <nav className="mt-16 mb-16">
-            <ul className="flex justify-center space-x-4">
-                <li className={`text-gray-400 ${currentPage === 1 ? "disabled" : "cursor-pointer text-gray-500"}`}>
-                    <span onClick={prevPage}>
-                        <Arrow direction={"left"} />
-                        Previous
-                    </span>
-                </li>
+            <ul className="flex justify-center items-center space-x-4">
+                <PaginationNavigationButton
+                    isDisabled={currentPage === 1}
+                    onClick={prevPage}
+                    label={"Previous"}
+                    arrowReversed={false}
```

Comment:
Instead of calling this `arrowReversed`, could you keep the arrow direction prop? This prop is used not only for left and right, but also up and down.

Reply:
Got it

Reply:
One question, @laushinka so for "up" and "down" arrowDirection, do I need to handle code for those too in `PaginationNavigationButtons` component. Cause right now the code flow would be like this?

![image](https://user-images.githubusercontent.com/28431356/199294020-3e096f69-8d36-4680-8de6-042e55a66476.png)

Reply:
You don't have to handle "up" and "down" here because we won't see those in Pagination. The intention is to be consistent with what the `<Arrow />` [component](https://github.com/gitpod-io/gitpod/blob/e0579ef9388cee0c4c7a2c85588683167e0cb0f7/components/dashboard/src/components/Arrow.tsx) expects :)

Reply:
Understood, I have made the respective changes

Reply:
Thanks! Could you squash and rebase please? :)